src/test/java/com/documents/TransportRequestDocumentRequestServiceTest.java

```diff
@@ -0,0 +1,193 @@
+
+        transportRequestDocumentToSave = transportRequestDocumentServiceImpl.findById((long) 123);
+
+        assertNotNull(transportRequestDocumentToSave);
+    }
+
+    @Test
+    public void functionality_transportRequest_findById_should_return_transportRequest() throws Exception {
+
+        TransportRequestDocument transportRequestDocumentToFind = new TransportRequestDocument();
+
+        setup(transportRequestDocumentToFind);
+
+        when(transportRequestDocumentRepository.findOne(any(long.class))).thenReturn(transportRequestDocumentToFind);
+
+        TransportRequestDocument findedTransportRequestDocument = transportRequestDocumentServiceImpl.findById((long) 3);
+
+        assertEquals(Long.valueOf(23), findedTransportRequestDocument.getId());
+        assertEquals("Transport", findedTransportRequestDocument.getDocumentName());
+        assertEquals("Ionica", findedTransportRequestDocument.getStudentName());
+        assertEquals("1234", findedTransportRequestDocument.getNrSeriesId());
+        assertEquals("2", findedTransportRequestDocument.getYearOfStudy());
+
+    }
+
+    @Test
+    public void behavioural_transportRequest_deleteById_should_return_true() throws Exception {
+
+        //Act
+        TransportRequestDocument transportRequestDocument = new TransportRequestDocument();
+
+        transportRequestDocument.setId((long) 3);
+
+        transportRequestDocumentServiceImpl.delete(transportRequestDocument.getId());
+
+        TransportRequestDocument findedTransportRequestDocument = transportRequestDocumentServiceImpl.findById((long) 3);
+
+        assertNull(findedTransportRequestDocument);
+
+    }
+
+
+    @Test
+    public void functionality_transportRequest_deleteById_should_return_delete_transportRequest() throws Exception {
+
+        //Act
+        TransportRequestDocument transportRequestDocument = new TransportRequestDocument();
+
+        transportRequestDocumentServiceImpl.delete(transportRequestDocument.getId());
+
+        //Assert
+        verify(transportRequestDocumentRepository).delete(transportRequestDocument.getId());
+
+    }
+
+    @Test
+    public void behavioural_transportRequest_findAll_should_return_true() throws Exception {
+
+        //Act
+        List<TransportRequestDocument> transportRequestDocuments = new ArrayList<>();
+
+        TransportRequestDocument transportRequestDocument = new TransportRequestDocument();
+
+        setup(transportRequestDocument);
+
+        when(transportRequestDocumentRepository.findAll()).thenReturn(transportRequestDocuments);
+
+        transportRequestDocuments.add(transportRequestDocument);
+
+        List<TransportRequestDocument> findedForms;
+        findedForms = transportRequestDocumentServiceImpl.findAll();
+
+        //Assert
+        assertNotNull(findedForms);
+
+    }
+
+    @Test
+    public void functionality_transportRequest_findAll_should_return_list_of_transportRequest() throws Exception {
+
```

Comment:
you could have left the comments (//Act and //Assert)...but if you don't want to , it's okay